Source/projections/Store/IProjectionOf.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { Cancellation } from '@dolittle/sdk.resilience';
+import { Key } from '../Key';
```

Comment:
```suggestion
import { Cancellation } from '@dolittle/sdk.resilience';

import { Key } from '../Key';
```

I like to separate the package and internal imports a little bit.